docker-compose.yml

```diff
@@ -37,6 +37,7 @@ services:
       - --db-schema=app
       - --workers=4
       - --batch-size=30
+      - --unsafe
       # - --log-level=debug
```

Comment:
Why `unsafe`? Isn't it unsafe? :D

Reply:
subql uses a restricted version of node to create a controlled env for running the code. sort of like endojs for hardened js

Therefore a lot of functionality (such as making api calls) is restricted. to enable it you need to run it in unsafe mode. you can read more about it [here](https://academy.subquery.network/indexer/run_publish/references.html#unsafe-node-service)